### ISSUE TITLE
Potential data sources/partners

### ISSUE THREAD

**Issue by meetar** [polygon-city/building-database#48]
- Toronto "3D Massing" data: http://www1.toronto.ca/wps/portal/contentonly?vgnextoid=d431d477f9a3a410VgnVCM10000071d60f89RCRD&vgnextchannel=1a66e03bb8d1e310VgnVCM10000071d60f89RCRD

- http://www.virtualcitysystems.de/en/solutions.html#3d-sdi apparently has some sample data at http://www.3dcitydb.org/3dcitydb/downloads/

**Comment by robhawkes** [comment]
I'm keen to explore the 3D City DB / CityGML stuff though I don't really know how widespread it is / how easy it is to get stuff out of the system.

**Comment by robhawkes** [comment]
I've heard back from Toronto and they are experiencing difficulties converting from DGN to OBJ. Looks like there's definitely a need for #43!

**Comment by meetar** [comment]
CityGML looks to be most common in Germany, added examples above

**Comment by robhawkes** [comment]
The Toronto models are released under the [Toronto Open Government License](http://www1.toronto.ca/wps/portal/contentonly?vgnextoid=4a37e03bb8d1e310VgnVCM10000071d60f89RCRD), which implies that it's ok to use and redistribute the buildings.

**Comment by robhawkes** [comment]
Looks like Berlin [are about to release](http://openstage.mipim.com/forums/270115-mipim-open-stage/suggestions/6816990-berlin-3d-goes-open-data-does-a-city-wide-3d-mod) their [3D city model](http://www.businesslocationcenter.de/en/berlin-economic-atlas/the-project) as open data in the coming month(s). I've reached out for more information.

**Comment by robhawkes** [comment]
I found out that the Netherlands are looking to release 3D data on their entire country under an open CC license. The buildings seem to be a OSM-level of detail but there may be potential for something higher detail: https://www.pdok.nl/nl/producten/pdok-downloads/basis-registratie-topografie/top10nl-3d

**Comment by robhawkes** [comment]
The city of Lyon in France has released a detailed CityGML 3D model under an open data license: http://www.opengeospatial.org/blog/2183?utm_source=twitterfeed&utm_medium=twitter

**Comment by robhawkes** [comment]
- [Hamburg, Montreal and Rotterdam CityGML models](http://www.citygmlwiki.org/index.php/Open_Data_Initiatives) released
- [Linz, Austria CityGML model](http://geo.data.linz.gv.at/katalog/geodata/3d_geo_daten_lod2/)
- [Bordeaux 3DS models](http://data.bordeaux-metropole.fr/data.php?themes=1), possibly sourced from CityGML

**Comment by robhawkes** [comment]
The Copenhagen model has been released, though I'm unsure of the license just yet: http://data.kk.dk/dataset/bymodel-3d-kobenhavn

**Comment by robhawkes** [comment]
Berlin have released their 3D city model in various formats (CityGML, 3DS, KMZ, etc), including a Java-based tool for downloading specific areas or even individual buildings. You can find it all here: http://www.businesslocationcenter.de/en/downloadportal

I've been told it's open data and unrestricted in use but I'm currently waiting on clarification on the exact license it has been released under.

**Comment by meetar** [comment]
Including textures! o_0

**Comment by robhawkes** [comment]
What's cool about these CityGML releases is that nearly all of them come with textures of some sort, it's great! The only reasons I've not been using them are because it dramatically bumps up the file-size and I haven't added material support to the bulk KML uploader just yet.

**Comment by robhawkes** [comment]
Here are a bunch of cities using 3DCityDB / CityGML: http://www.3dcitydb.net/3dcitydb/3dcitydb-in-action/

**Comment by robhawkes** [comment]
We may have Brussels if everything goes to plan – I have the CityGML data but it's currently broken and I'm hoping to work it out with them.

**Comment by robhawkes** [comment]
I managed to get the Brussels CityGML data working (sample added to the dev site), though a lot of the collada models have missing faces or flipped normals in places. I've reached out to the CityGML guys to see if there's a way around this, as the CityGML file is valid so it must be due to the collada export.

**Comment by meetar** [comment]
Berlin districts: ftp://download-berlin3d.virtualcitymap.de/citygml/

**Comment by bcamper** [comment]
This project was passed along to me: http://www.gis.bgu.tum.de/en/projects/new-york-city-3d/

It's a CityGML model of NYC, derived from multiple open sources. It's unclear how much more or less complete it is (in model detail and metadata) than just using OSM (for example, it's not going to have any of the OSM tags, which we use for styling in Tangram), but it could be another fruitful test dataset, and one we could even consider importing. (Ideally there would be an official NYC government model but  don't know what the timeline is for that.)

**Comment by meetar** [comment]
Looks like it's strictly building footprints extruded to the tallest height of the building, judging by this view of the Empire State Building in their Cesium viewer (bit.ly/1F8MwJO):

<img width="683" alt="screen shot 2015-10-12 at 11 33 05 am" src="https://cloud.githubusercontent.com/assets/459970/10431829/5e3ddff2-70d5-11e5-86dc-86068ed0f415.png">

**Comment by robhawkes** [comment]
Yeah that NYC CityGML stuff is LOD1, which is basically an extruded footprint. The benefit we may have in this case is that the footprints from the NYC portal (which I believe these are based on) could be more accurate across the city compared to OSM (due to hand-drawn errors, etc). On the flip-side, we won't benefit from the quirky things in the OSM data like the faux-3D detail via the overuse of building parts and 3D tags (eg. the madness that is One World Trade Center).

There's nothing stopping us uploading this as an alternative NYC dataset though! Something to consider for v2 would be situations like this where we may have multiple versions of a city, each with a benefit. The browsing / search / tile interface wouldn't handle this very well right now.

**Comment by bcamper** [comment]
Makes sense, just wanted to note and we can revisit later in case an OSM
alternative is desirable.
